src/197F40.c

```diff
@@ -2618,11 +2618,134 @@ ApiStatus ResetAllActorSounds(Evt* script, s32 isInitialCall) {
     return ApiStatus_DONE2;
 }
 
-INCLUDE_ASM(s32, "197F40", SetActorSounds);
+s32 SetActorSounds(Evt* script, s32 isInitialCall) {
+    Bytecode* args = script->ptrReadPos;
+    s32 actorID = evt_get_variable(script, *args++);
+    Actor* actor;
+    s32 temp_s0_4;
+    s32 temp_s1;
+    s32 temp_s3;
+
+    if (actorID == ACTOR_SELF) {
+        actorID = script->owner1.actorID;
+    }
+
+    temp_s3 = evt_get_variable(script, *args++);
+    temp_s0_4 = evt_get_variable(script, *args++);
+    temp_s1 = evt_get_variable(script, *args++);
+    actor = get_actor(actorID);
+
+    switch (temp_s3) {
+        case 0:
+            actor->actorTypeData1[0] = temp_s0_4;
+            actor->actorTypeData1[1] = temp_s1;
+            break;
+        case 1:
+            actor->actorTypeData1[2] = temp_s0_4;
+            actor->actorTypeData1[3] = temp_s1;
+            break;
+        case 2:
+            actor->actorTypeData1[4] = temp_s0_4;
+            break;
+        case 3:
+            actor->actorTypeData1[5] = temp_s0_4;
+            break;
+        case 4:
+            actor->actorTypeData1b[0] = temp_s0_4;
+            break;
+        case 5:
+            actor->actorTypeData1b[1] = temp_s0_4;
+            break;
+    }
+
+    return ApiStatus_DONE2;
+}
+
+ApiStatus ResetActorSounds(Evt* script, s32 isInitialCall) {
+    Bytecode* args = script->ptrReadPos;
+    s32 actorID = evt_get_variable(script, *args++);
+    Actor* actor;
+    s32 temp_s3;
+
+    if (actorID == ACTOR_SELF) {
+        actorID = script->owner1.actorID;
+    }
+
+    temp_s3 = evt_get_variable(script, *args++);
+    evt_get_variable(script, *args++);
+    evt_get_variable(script, *args++);
```

Comment:
Please save these in variables, even tho they are unused. I'd guess someone copied above function and rewrote it but then forgot to remove these